packages/auto-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wxt-dev/auto-icons",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Oh wait, don't manually bump the version, this is handled in the release workflow.

Reply:
You're good! I should have caught this in review. When a project uses conventional commits, and often times with other release methods, there's usually a specific commit for bumping the version, so there's a nice clean change to apply a tag to. With conventional commits, release tools do the bumping automatically for you.